pkg/cerrl/cerrl.go

```diff
@@ -110,6 +116,16 @@ func run(pass *analysis.Pass) (interface{}, error) {
 	return nil, nil
 }
 
+func isHasSuffix(path string, list []string) bool {
+	for _, item := range list {
+		if strings.HasSuffix(path, item) {
```

Comment:
it will not work for strings.HasSuffix("wasfaty/pkg/cerror/v2", "/pkg/cerror/v*")
we don't need to search for exact `v*` instead need to check v2,v3,...

Reply:
Done